src/client/app/graph/graph.component.ts

```diff
@@ -45,42 +45,65 @@ export class GraphComponent implements OnInit, OnDestroy {
       let numCards = model.getRoot().get('cards').length;
 
       let card1 = model.create(Card);
-      card1.position = {x: 10 + numCards * 5, y: 10 + numCards * 5};
-      card1.text = 'card #' + numCards;
+      card1.position = {x: 10 + numCards * 20, y: 10 + numCards * 20};
+      card1.text = 'card #' + numCards++;
```

Comment:
Ehhhh, postincrement operators in the middle of a line is surprising code, imo. I'd avoid it if possible.

Reply:
Agreed but this code is a temporary hack anyway and is getting removed in the next PR.